examples/ballerina-to-openapi/ballerina_to_openapi.description

```diff
@@ -1,4 +1,4 @@
 // The Ballerina to OpenApi tool is used to generate an OpenAPI Specification(OAS) v3 definition of a Ballerina service.
 // This tool's command structure is as below.
-// `ballerina openapi gen-contract [<module>:]<service-name> [-i <ballerina-file>] [-o <contract-file>] [-s|--skip-bind] [-h|--help]`<br/><br/>
+// `ballerina openapi  [-i <ballerina-file>] [-o <contract-file>] `<br/><br/>
```

Comment:
Need to use ```bash for commands and ```ballerina for Ballerina files. Please check and update all occurrences.